docs/src/main/asciidoc/spring-cloud-commons.adoc

```diff
@@ -995,6 +995,28 @@ public class CustomLoadBalancerConfiguration {
 
 WARNING: `HealthCheckServiceInstanceListSupplier` has its own caching mechanism based on Reactor Flux `replay()`. Therefore, if it's being used, you may want to skip wrapping that supplier with `CachingServiceInstanceListSupplier`.
 
+=== Same instance preference for LoadBalancer
+
+It is possible to set up the LoadBalancer in such a way that it will prefer the instance that was previously selected, if that instance is available.
```

Comment:
Change to:

You can set up the `LoadBalancer` in such a way that it prefers the instance that was previously selected, if that instance is available.